_sass/pages/_partners.scss

```diff
@@ -153,5 +153,7 @@
   display: inline-flex;
   flex-shrink: 0;
   font-size: 0.81rem;
+  height: 2rem;
+  margin-top: 0.5rem;
```

Comment:
I don't see anything in the pull request or ticket about these changes. I'm assuming the idea is to try to force the consistent height alignment of the buttons in the header?

Reply:
Yes that is correct.